fix #11 - wrap ajax in Promises

groovecoder opened the pull request:
<!---
@huboard:{"order":12.0,"milestone_order":12,"custom_state":""}
-->

groovecoder committed:
fix #11 - wrap ajax in Promises

openjck commented:
Just a question about workflow. Should the _.xpi_ be re-built with this commit or is the idea to continually change the source and re-build the package as part of a special release/deploy process? Either way is fine. Just want to be sure to know what to look for.

openjck commented:
Didn't mean to close. Reopening...

openjck commented on data/doiuse-script.js:
```diff
@@ -1,42 +1,51 @@
+}
+
+function postToDoiuse (css) {
+  var styleSheetContent = css;
```
Not a big deal, but why the rename?

openjck commented:
Also, this is very cool. I'm a fan.

groovecoder committed:
for #11 - updated xpi; keep function param name

groovecoder commented:
Updated.

Good point on the `.xpi` file. I updated the file in this PR since it's already in the repo. In the future, yes - we would compile the `.xpi` file as part of a build/deploy process, and (hopefully) automatically upload it to addons.mozilla.org.

openjck commented:
Looks good, works as expected r+
